content/api/commands/origin.md

```diff
@@ -0,0 +1,512 @@
+---
+title: origin
+---
+
+Visit multiple domains of different
+[origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#definition_of_an_origin)
+in a single test.
+
+In normal use, a single Cypress test may only run commands in a single origin, a
+limitation determined by standard web security features of the browser. The
+`cy.origin()` command allows your tests to bypass this limitation.
+
+<Alert type="warning">
+
+<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
+Experimental</strong>
+
+The `origin` API is currently experimental, and can be enabled by setting
+the [`experimentalSessionAndOrigin`](/guides/references/experiments) flag
+to `true` in the Cypress config.
+
+Enabling this flag does the following:
+
+- It adds the [`cy.session()`](/api/commands/session) and `cy.origin()`
+  commands, and [`Cypress.session`](/api/cypress-api/session) API.
+- It adds the following new behaviors (that will be the default in a future
+  major update of Cypress) at the beginning of each test:
+  - The page is cleared (by setting it to `about:blank`).
+  - All active session data (cookies, `localStorage` and `sessionStorage`)
+    across all domains are cleared.
+- It overrides
+  the [`Cypress.Cookies.preserveOnce()`](/api/cypress-api/cookies#Preserve-Once) and
+  [`Cypress.Cookies.defaults()`](/api/cypress-api/cookies#Defaults) methods.
```

Comment:
What does it override to? Do users need to change their tests because of this?

Reply:
Maybe better, maybe not, but this is the second question we have gotten about this line.
```suggestion
- It supersedes
  the [`Cypress.Cookies.preserveOnce()`](/api/cypress-api/cookies#Preserve-Once) and
  [`Cypress.Cookies.defaults()`](/api/cypress-api/cookies#Defaults) methods.
```